### ISSUE TITLE
Resources: Pagination button must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by RyanAJensen** [pulibrary/DSS#565]
### Steps to recreate
Visit /resources and run the Axe dev tools

### URL
http://localhost:3000/resources

### Issues
"Previous" pagination button does not meet minimum color contrast ratio threshold

### Severity
High

### WCAG Success Criterion
- [ ]  Elements must meet minimum color contrast ratio thresholds

### WCAG Conformance level (A, AA, AAA)
AA

### User impact
Some people cannot read faint text. For all consumers of visual content, adequate light-dark contrast is needed between the relative luminance of text and its background for good readability. 

### Recommended fix
Element has insufficient color contrast of 1.21 (foreground color: #ebe9e2, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] WCAG Success Criterion are resolved
